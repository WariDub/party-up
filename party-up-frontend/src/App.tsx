import React from 'react';
// import ProfilePage from './Profile/ProfilePage';
import SignUp from './Auth/SignUpForm';
// import SearchPage from './Search/SearchPage';

function App(): JSX.Element {
  return (
    // <ProfilePage />
    // <SearchPage />
    <SignUp />
  );
}

export default App;
