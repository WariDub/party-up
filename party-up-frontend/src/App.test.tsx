import React from 'react';
import { render, screen } from '@testing-library/react';
import App from './App';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';


describe('App', () => {
  test('It should mount', () => {
    let component = shallow(<App/>);

    expect(component.length).toBe(1);
  });

  test('It should match snapshot of App, to ensure that the UI is unintentially affected by any updates', () => {
    let component = renderer.create(<App/>).toJSON();

    expect(component).toMatchSnapshot();
  })
});