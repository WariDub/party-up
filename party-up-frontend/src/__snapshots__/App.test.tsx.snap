// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App It should match snapshot of App, to ensure that the UI is unintentially affected by any updates 1`] = `
<div
  className="wrapper"
>
  <div
    className="login-wrap"
  >
    <div
      className="login-html"
    >
      <input
        className="sign-in"
        id="tab-1"
        name="tab"
        type="radio"
      />
      <label
        className="tab"
        htmlFor="tab-1"
      >
        Sign In
      </label>
      <input
        className="sign-up"
        id="tab-2"
        name="tab"
        type="radio"
      />
      <label
        className="tab"
        htmlFor="tab-2"
      >
        Sign Up
      </label>
      <div
        className="login-form"
      >
        <div
          className="sign-in-htm"
        >
          <div
            className="group"
          >
            <label
              className="label"
              htmlFor="user"
            >
              Username
            </label>
            <input
              className="input"
              id="user"
              onChange={[Function]}
              type="text"
              value=""
            />
          </div>
          <div
            className="group"
          >
            <label
              className="label"
              htmlFor="pass"
            >
              Password
            </label>
            <input
              className="input"
              data-type="password"
              id="pass"
              onChange={[Function]}
              type="password"
              value=""
            />
          </div>
          <div
            className="group"
          >
            <input
              checked={true}
              className="check"
              id="check"
              type="checkbox"
            />
            <label
              htmlFor="check"
            >
              <span
                className="icon"
              />
               Keep me Signed in
            </label>
          </div>
          <div
            className="group"
          >
            <button
              className="button"
              onClick={[Function]}
              type="submit"
              value="Sign In"
            >
              Sign In
            </button>
          </div>
          <div
            className="hr"
          />
          <div
            className="foot-lnk"
          >
            <a
              href="#forgot"
            >
              Forgot Password?
            </a>
          </div>
        </div>
        <div
          className="sign-up-htm"
        >
          <div
            className="group"
          >
            <label
              className="label"
              htmlFor="user"
            >
              Username
            </label>
            <input
              className="input"
              id="user"
              onChange={[Function]}
              type="text"
              value=""
            />
          </div>
          <div
            className="group"
          >
            <label
              className="label"
              htmlFor="pass"
            >
              Password
            </label>
            <input
              className="input"
              data-type="password"
              id="pass"
              onChange={[Function]}
              type="password"
              value=""
            />
          </div>
          <div
            className="group"
          >
            <label
              className="label"
              htmlFor="pass"
            >
              Repeat Password
            </label>
            <input
              className="input"
              data-type="password"
              id="pass"
              onChange={[Function]}
              type="password"
              value=""
            />
          </div>
          <div
            className="group"
          >
            <label
              className="label"
              htmlFor="pass"
            >
              Email Address
            </label>
            <input
              className="input"
              id="pass"
              onChange={[Function]}
              type="text"
              value=""
            />
          </div>
          <div
            className="group"
          >
            <button
              className="button"
              onClick={[Function]}
              type="submit"
              value="Sign Up"
            >
              Register
            </button>
          </div>
          <div
            className="hr"
          />
          <div
            className="foot-lnk"
          >
            <label
              htmlFor="tab-1"
            >
              Already Member?
            </label>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
